### DISCUSSION TITLE
Can I host Supabase on my own server or it's only Saas?

### DISCUSSION THREAD

**Discussion by frederikhors** [supabase/supabase#2813] [Questions] [answered]
Can I host Supabase on my own server or it's only Saas?

**Comment by soedirgo** [comment, marked as answer]
Hey @frederikhors, this has been answered here: https://github.com/supabase/supabase/discussions/2218.

TLDR, you can self host with a few caveats, in particular we're still working on the open source dashboard which will replace the current (closed source) one.
